extensions/adobe/experience/target/optionevent.schema.json

```diff
@@ -0,0 +1,88 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/target/activity/activityevent/optionevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Activity Option Event",
+  "type": "object",
+  "description": "Represents Activity Option Event.",
+  "definitions": {
+    "optionevent": {
+      "properties": {
+        "https://ns.adobe.com/experience/target/activity/activityevent/optionevent/location": {
+          "title": "Location",
```

Comment:
More specific title please

Reply:
All target users are used to using the "location" in the context of activities.

Reply:
that may be, but it conflicts with our use of location elsewhere in XDM.  Maybe call it an activityLocation?

Reply:
sure